custom_components/frigate/services.yaml

```diff
@@ -0,0 +1,20 @@
+---
+favorite_event:
```

Comment:
Opinion: Should we call this `retain_event`. That's what it's call in all the other APIs.

Reply:
I was going back and forth on this, but as this would be user facing, I think `favorite` is more straightforward to a user

Reply:
Yeah, me too. The counterargument would be that it's not actually _that_ user facing, it's just another API really. Also, very minor, 'favorite' varies in spelling between different English variants, 'retain' does not.

`favorite_event` SGTM since we're both on the same fence.

Reply:
Would it be weird if I set the service as `retain_event` but left the `name` as Favorite Event ? The name part is the user facing part in this since it shows in the service dropdown selector

Reply:
Probably (would be weird). Might as well go all in on `favorite` ;-)

Reply:
👌